File: PHP/Iniciante/1012.php
<?php

list($a, $b, $c) = explode (" ", readline());

$a = floatval($a);
$b = floatval($b);
$c = floatval($c);
$pi = 3.14159;

$triangulo = number_format(($a * $c) / 2, 3, '.', '');
$circulo = number_format($pi * pow($c, 2), 3, '.', '');
$trapezio = number_format(($a + $b) * $c / 2, 3, '.', '');
$quadrado = number_format($b * $b, 3, '.', '');
$retangulo = number_format($a * $b, 3, '.', '');

echo "TRIANGULO: $triangulo\n";
echo "CIRCULO: $circulo\n";
echo "TRAPEZIO: $trapezio\n";
echo "QUADRADO: $quadrado\n";
echo "RETANGULO: $retangulo\n";

  
?>

File: C++/Iniciante/1012.cpp
#include <stdio.h>
#include <iostream>
#include <cmath>

int main() {
  double a, b, c, pi = 3.14159;
  std::cin >> a >> b >> c;
  
  double triangulo = (a * c) / 2.0;
  double circulo = pi * (std::pow(c, 2));
  double trapezio = (a + b) * c / 2.0;
  double quadrado = (std::pow(b, 2));
  double retangulo = (a * b);

  printf("TRIANGULO: %.3lf\n", triangulo);
  printf("CIRCULO: %.3lf\n", circulo);
  printf("TRAPEZIO: %.3lf\n", trapezio);
  printf("QUADRADO: %.3lf\n", quadrado);
  printf("RETANGULO: %.3lf\n", retangulo);
  
}
